backend-services/voting-app/src/main/java/org/cardano/foundation/voting/config/SpringSecurityConfiguration.java

```diff
@@ -90,7 +90,6 @@ public SecurityFilterChain filterChain(HttpSecurity http) throws Exception {
                     .requestMatchers(new AntPathRequestMatcher("/api/vote/receipt", GET.name())).authenticated()
                     .requestMatchers(new AntPathRequestMatcher("/api/vote/receipt", HEAD.name())).authenticated()
                     .requestMatchers(new AntPathRequestMatcher("/api/vote/receipts", GET.name())).authenticated()
-                    .requestMatchers(new AntPathRequestMatcher("/api/vote/receipts", HEAD.name())).authenticated()
```

Comment:
Would make sense also to remove {HEAD } from the controller

Reply:
done!